### PR TITLE
Submitting  Intuiter version 0.7.3.yaml

### PR DESCRIPTION
**Pull request by seonglae**
- [o] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [o] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [o] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [o] Have you tested your manifest locally with `winge@mast install --manifest <path>`?
- [o] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22925)

### CODE REVIEW

File: manifests/s/Seonglae/Intuiter/0.7.3/Seonglae.Intuiter.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Publisher: Seonglae
+PackageName: Intuiter
+PackageVersion: 0.7.3
+License: MIT
+InstallerType: exe
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+Installers:
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/s/Seonglae/Intuiter/0.7.3/Seonglae.Intuiter.yaml:11]:
```suggestion
- Architecture: x64
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

File: manifests/s/Seonglae/Intuiter/0.7.3/Seonglae.Intuiter.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+PackageIdentifier: Seonglae.Intuiter
+Publisher: Seonglae
```
Comment (ItzLevvie) [inline manifests/s/Seonglae/Intuiter/0.7.3/Seonglae.Intuiter.yaml:2]:
```suggestion
Publisher: Seonglae Cho
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by seonglae** [comment]
@msftbot

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**